arches/app/models/resource.py

```diff
@@ -238,6 +259,46 @@ def get_flattened_tiles(self):
             tiles.extend(tile.get_flattened_tiles())
         return tiles
 
+    @staticmethod
+    def bulk_update(resources, transaction_id=None):
```

Comment:
@aarongundel, I don't see bulk_update called anywhere

Reply:
This method seems an almost duplicate of the "bulk_save" method.  This one, though, isn't calling a "bulk" django method to update tiles and instead calls "update_or_create".  There is a django provided "bulk_update" method that could be used here although in reality it seems this whole method could just be subsumed by our "bulk_save" method with some creative restructuring.

Reply:
Yeah, I think it would be good to either move the code into the project where it is used, or add the update logic to the bulk_save.